padinfo/padinfo.py

```diff
@@ -611,6 +619,32 @@ async def leaderskillsingle(self, ctx, *, query):
 
         await self._do_menu(ctx, self.ls_emoji, EmojiUpdater(emoji_to_embed))
 
+    @commands.command(aliases=['tfinfo', 'xforminfo'])
+    @checks.bot_has_permissions(embed_links=True)
+    async def transforminfo(self, ctx, *, query):
+        """Show info about a transform card, including some helpful details about the base card."""
+        beta_id3 = await self.config.user(ctx.author).beta_id3()
+        dgcog = await self.get_dgcog()
+        bm, err, debug_info, tfm, base_rarity, acquire_raw, true_evo_type_raw = \
+            await perform_transforminfo_query(dgcog, query, beta_id3)
+        
+        if not tfm:
+            await ctx.send('Your query (`{}`) did not find a monster that transforms.'.format(query))
```

Comment:
So this should probably have two non-error failure messages: First case is that there's no monster matched at all, in which case "sorry no monster matched this" and second case is "sorry no transforming monster matched this" - so maybe first check if not base_mon then if not transformed_mon, and have `perform_transforminfo_query` support both of these possible return values

(the first case would simply call our normal no-result error function)